#mela by se vyhodit chyba kvuli carce za poslednim parametrem
def func(x,y,):
    x = y

func(2, 5)